markhor_tracks/include/markhor_hw_interface.hpp

```diff
@@ -69,14 +69,16 @@ class MarkhorHWInterface : public hardware_interface::RobotHW
 
   double tracks_fb_coeff = 1;
 
+  double track_encoder_to_rad_coeff = 39384;
```

Comment:
Could be a constant, like the one on line 36

Reply:
Can't be a constant, this is the default value, this variable is overwritten by the parameter set in the launchfile